bw-dev

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# exit on errors
+set -e
+
+CMD=$1
+if [ -n "$CMD" ]; then
+    shift
+fi
+
+# show commands as they're executed
+set -x
+
+case "$CMD" in
+    site:compile)
+        python3 generate.py
+        ;;
+    messages:generate)
+        mkdir -p locale/$@/LC_MESSAGES && touch locale/$@/LC_MESSAGES/messages.po
+        xgettext --no-wrap --join-existing --from-code=UTF-8 --keyword=_ --output=locale/$@/LC_MESSAGES/messages.po `find templates -name "*.html"`
+        sed -i "" "s/Content-Type: text\/plain; charset=CHARSET/Content-Type: text\/plain; charset=UTF-8/g" locale/$@/LC_MESSAGES/messages.po
```

Comment:
This does not run on my system (archlinux). Should it be `sed -i -e "s/Content-Type: text\/plain; charset=CHARSET/Content-Type: text\/plain; charset=UTF-8/g" locale/$@/LC_MESSAGES/messages.po`?

The error message reads:
```
sed: can't read s/Content-Type: text\/plain; charset=CHARSET/Content-Type: text\/plain; charset=UTF-8/g: No such file or directory
```

Reply:
You're right! My guess is that my version of sed on macOS doesn't work the same way as your version on Linux. I'm all for using the Linux-compatible command. Thanks for the fix!

Reply:
(apart from that, did you have any trouble with the process and commands for translating the website? I made it all quickly, so there might be ways to make it better)

Reply:
I noticed several warnings from `xgettext` and the slow generation process because of re-requesting `nodeinfo`.
I tried making some improvements in my forked repository at https://github.com/GuDzpoz/join-bookwyrm/commit/7bc65adc749e30e447c6ff6a344ece1705a618b2 .

(Also, is `bookclub.techstartups.space` down? In the above commit I simply set its user count to `NaN` and open_registration to False, but we should probably have a different template (or styling? or a message?) for unreachable sites.

Reply:
thank you! I do think that instance is down, I'll remove it from the page

Reply:
can you make a PR for those changes? they're great :)

Reply:
PR opened: #8 ;)